### ISSUE TITLE
[Feature Request] Manage and Review Pull Requests

### ISSUE THREAD

**Issue by christianbraeunlich** [tfenster/azdevops-vscode-simplify#33]
Hi, thanks a lot for this great VS Code extension! Really comes in handy to view my assigned work items.

Wouldn't it be nice if we could stay in VS Code during the code review / managing PRs?

You plan to work on this feature or are there other solutions we could use? I already tried https://github.com/ankitbko/vscode-pull-request-azdo but I couldn't Sign In to Azure Devops via this extension.

Thanks in advance.

**Comment by tfenster** [comment]
Thanks for the feedback, very much appreciated 😊

For your question: Yes, it would be nice, but in my opinion, that would be quite a lot of work (I would guess a lot more than we currently have in total) and I personally won't be able to do something on that scale in the foreseeable future. Maybe @DavidFeldhoff feels differently, but otherwise I'd put it as something for our long-term backlog

**Comment by DavidFeldhoff** [comment]
I just had a look at the other extension and building something like that on our own seems to be quite tough. Simply comparing two branches should be possible somehow, but having the discussions and similar things as well in VS Code (and I suggest it should be synced with the discussions on the DevOps PR) seems to be too much for the time I currently can spent on this one, so I'd agree with you @tfenster .

**Comment by Trass3r** [comment]
Yeah I wouldn't even bother, MS should add that Azure support from the fork to their GitHub PR extension instead.
In the meantime for me it would be enough to have a button with the number of open PRs and a link to the corresponding website.
Similarly there should be a button with the build status of the current branch and a link to the pipeline page. But I guess that should be its own issue.
